test/dns/query.cpp

```diff
@@ -99,6 +104,18 @@ TEST_CASE("throw error while fails evdns_set_options for attempts") {
     REQUIRE(false);
 }
 
+TEST_CASE("throw error while fails evdns_set_options for timeout") {
+    try {
+        create_evdns_base<::evdns_base_new, ::evdns_base_nameserver_ip_add,
+                          base_free_evdns_set_options_timeout>(
+            {{"dns/attempts", "nexa"}}, Reactor::global());
+    } catch (std::runtime_error &) {
+        REQUIRE(base_free_evdns_set_options_timeout_flag);
+        return;
+    }
```

Comment:
Can you try by wrapping the template invocation with `( ... )` as suggested above? I think I encountered this problem when developing NDT and fixed it using one extra layer of braces.